clients/task/task.go

```diff
@@ -52,9 +54,217 @@ func (c Client) Create(ctx context.Context, params *CreateParams) error {
 	})
 }
 
+type FindParams struct {
+	clients.OrgParams
+	TaskID string
+	UserID string
+	Limit  int
+}
+
+func (c Client) Find(ctx context.Context, params *FindParams) error {
+	org, err := c.getOrg(&params.OrgParams)
+	if err != nil {
+		return err
+	}
+	if params.Limit < 1 {
+		return fmt.Errorf("must specify a positive limit, not %d", params.Limit)
+	}
+
+	var tasks []api.Task
+	// If we get an id, just find the one task
+	if params.TaskID != "" {
+		task, err := c.GetTasksID(ctx, params.TaskID).Execute()
+		if err != nil {
+			return err
+		}
+		tasks = append(tasks, task)
+	} else {
+		// filter on all tasks
+		taskGet := c.GetTasks(ctx).Org(org).Limit(int32(params.Limit))
```

Comment:
We could add `format: int64` to the schema for the limit parameter if we want the codegen to generate that. It looks like the hand-written server code uses `int` so (I think) it should be safe: https://github.com/influxdata/influxdb/blob/master/paging.go#L33-L34

If we want to keep the API as-is for now, could you move the conversion out a level and type `Limit` as an `int32` in `FindParams`?

Reply:
I can't move the conversion out, and I think int64 is way overkill for these limits - for now I'll check and error if the value doesn't fit in 32 bits, before conversion.